objectRecognitionSource/README.md

```diff
@@ -279,6 +279,103 @@ Options available for all Queries (not prepended by anything) are:
     instead of only the objects recognized. Note that the data will be the sole occupant of a 1-element array when
     received, instead of being immediately available as a JSON object. This is because Query results are mandated
     to be arrays. Default is false.
+    
+Unique query capabilities available for the YOLO Processor are as follows:
+
+*   **NOTE:** All YOLO Processor queries should have an "operation" parameter specified. If this parameter is not specified, 
+it will be set to the default value, "processNextFrame".
+
+*   **Upload images to VANTIQ:**
+    *   The user can specify an image, or multiple images to be uploaded to VANTIQ as a document. The images will be those 
+    that are saved in the output directory, which is defined in the source configuration.
+    *   Parameters:
+        *   "operation": Required. Must be set to "upload".
+        *   *You should specify exactly one of the following two values. If neither is specified, a query error will be 
+        returned. If both values are specified, the imageName value will be used.*
+            *   "imageName": A string value representing the name of the file to be uploaded.
+            *   "imageDate": A list containing two strings, a start date and an end date. All locally saved images falling 
+            between the start and end date, (inclusive), will be uploaded. Dates must be formatted in the following 
+            manner: "yyyy-MM-dd--HH-mm-ss".
+                *   To select all images *before* or *after* a certain date, the "-" value can be used as one of the date 
+                strings in the list. For example, the following would save all dates *before* the given date: 
+                    *   \["-", yourEndDate\]. 
+                *   To select *all files* in the output directory, one could use the following value for imageDate: 
+                    *   \["-", "-"\].
+        *   "savedResolution": Optional. This value can be set in the exact same way as it is set in the source configuration. 
+        If it is defined here as a query parameter, it will override the value set in the source configuration.
```

Comment:
One thing I was thinking of adding to the README, but not sure if it's worth writing out. Say someone set their source configuration to save images locally, with a longEdge of 600. The images would already be resized, and so you would only be able to set a longEdge less than the initial 600 from the source configuration.

Does that make sense? I'm wondering if I should put this in writing somewhere in the README, or if it is already implied since we said earlier that we will not support enlarging images.

Reply:
I think it makes sense to add.  I think it's hard to explain all the ins & outs, so probably something like "the setting cannot be larger than that provided in the source configuration (since that defines how the images are saved)."  That gets to what's real without having to explain the various machinations that may go on...